sdk/keyvault/keyvault-admin/samples-dev/snippets/_prerequisites.md

```diff
@@ -1,3 +1,3 @@
 To quickly create the needed Key Vault Managed HSM resources in Azure and to receive a connection string for them, you can deploy our sample template by clicking:
 
-[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
\ No newline at end of file
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.bicep)
```

Comment:
```suggestion
[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2F0312603aff3732fb39aa8c2e4aa2d31049f3c212%2Fsdk%2Fkeyvault%2Ftest-resources.bicep)
```

Hmm unfortunately I am not sure this works with bicep files - only arm.json ... don't want to block you on this though as we can probably just update the snippets here (we probably should do so anyway and modernize it - seems to still use connection strings anyway). Feel free to make an issue for updating these, in the meanwhile maybe we can use a specific commit instead of just `master`? Same for the other snippets

Reply:
I see, I compiled bicep to arm json file and checked it back in to keep these links intact.